src/execution_plan/execution_plan_modify.c

```diff
@@ -159,7 +159,7 @@ void ExecutionPlan_DetachOp(OpBase *op) {
 OpBase *ExecutionPlan_LocateOpResolvingAlias(OpBase *root, const char *alias) {
 	if(!root) return NULL;
 
-	uint count = (root->modifies) ? array_len(root->modifies) : 0;
+	uint count = array_len(root->modifies);
```

Comment:
is `modifies` always not NULL?

Reply:
`modifies` can be NULL, but this check is redundant - the exact same ternary is found inside `array_len`.